src/List.js

```diff
@@ -16,8 +17,25 @@ const List = () => {
     emptyList = true;
   }
 
+  //When the user selects an item, search the items array to find the item selected.
+  //Use the item's last purchased date and the frequency to calculate the estimated
+  //next purchased date. Update the database with the new property items.
   const handleChange = async e => {
```

Comment:
Thanks for commenting your code! However, as Scott pointed out, it might make more sense to give a more meaningful name to the function instead of explaining what `handleChange` does.

Reply:
I changed `handleChange` to `handlePurchase` and `ingr` into `item`. I also removed the commented out code, but added a new line saying that our filter will only return one item as an array (hence the use of the index 0).